packages/x-components/src/components/base-dropdown.vue

```diff
@@ -5,11 +5,18 @@
     @keydown.up.prevent="highlightPreviousItem"
     :class="dropdownCSSClasses"
     class="x-dropdown"
+    role="combobox"
+    aria-haspopup="listbox"
+    tabIndex="0"
+    :aria-expanded="isOpen"
+    :aria-label="value"
```

Comment:
The aria label shouldn't be the selected value. It should be a description of what the field does. Moreover like I said below, the value can be an object. So adding this you are just making the component more inaccessible.

Reply:
Yep, my intention was to use it as fallback since there's no a common name descriptive enough I can come with both for the `base-column-picker-dropdown` and `sort-dropdown`. Any thoughts? I wouldn't trust on devs to remember to add it when creating their views ^^'

Reply:
I wouldn't force it here. In the end this is more like an "abstract" component, we don't have that info at this level. We can add it for the "concrete" implementation of this component (the column picker and the sort) like you did with the switch.